### PR TITLE
owners tab and adding owner no styling

### PR DESCRIPTION
**Pull request by namitamanohar**

### CODE REVIEW

File: DogWalkerMVC/Controllers/OwnerController.cs [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"SELECT o.[Name] AS OwnerName, n.[Name] AS 'Neighborhood Name', Count(d.[Name]) AS 'Number of Dogs'
+                        FROM Owner o
+                        LEFT JOIN Neighborhood n 
+                        ON n.Id = o.NeighborhoodId
+                        LEFT JOIN Dog d 
+                        ON d.OwnerId = o.Id 
+                        GROUP BY o.[Name], n.[Name]";
```
Comment (AdamSheaffer) [inline DogWalkerMVC/Controllers/OwnerController.cs:47]: Awesome! Query looks great. I'm thinking We'll probably also need the owner's `Id` too though. It's not immediately obvious from the mockup but if I'm looking at figma correctly, you can click an owner's name and it goes to that owner's detail page. Is that correct @brendalong ? 

@namitamanohar if that _is_ the case, we'll need the Owner's Id to be able to create that `<a>` tag

Comment (namitamanohar) [reply DogWalkerMVC/Controllers/OwnerController.cs:47]: Can this branch be merged? As we have not created the details view for the owners

File: DogWalkerMVC/Controllers/OwnerController.cs [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+
+        // GET: Owner/Create
+        public ActionResult Create()
+
+        {
+            var viewModel = new AddOwnerViewModel()
+            {
+                NeighborhoodOptions = GetNeighborhoodOptions()
+            };
+            return View(viewModel);
```
Comment (AdamSheaffer) [inline DogWalkerMVC/Controllers/OwnerController.cs:127]: 👍

**Review by AdamSheaffer — APPROVED** [review]
Looks good. Added a comment about including the Owner Id in the view model, but I'm ok with merging this for now.

File: DogWalkerMVC/Controllers/OwnerController.cs (hunk 1 above)
Comment (AdamSheaffer) [reply DogWalkerMVC/Controllers/OwnerController.cs:47]: Yeah I'm gonna go ahead and merge it. We'll probably eventually need to add that tho

**Pull request merged**